website/source/docs/providers/aws/r/rds_cluster_instance.html.markdown

```diff
@@ -71,6 +71,9 @@ enhanced monitoring metrics to CloudWatch Logs. You can find more information on
 what IAM permissions are needed to allow Enhanced Monitoring for RDS Instances.
 * `monitoring_interval` - (Optional) The interval, in seconds, between points when Enhanced Monitoring metrics are collected for the DB instance. To disable collecting Enhanced Monitoring metrics, specify 0. The default is 0. Valid Values: 0, 1, 5, 10, 15, 30, 60.
 * `promotion_tier` - (Optional) Default 0. Failover Priority setting on instance level. The reader who has lower tier has higher priority to get promoter to writer
+* `preferred_backup_window` - (Optional) The daily time range during which automated backups are created if automated backups are enabled.
```

Comment:
Suggest: adding a syntax example here like was done for `preferred_maintenance_window`